src/evaluators/utility/PHAL_UnivariateDistribution.hpp

```diff
@@ -0,0 +1,300 @@
+//*****************************************************************//
+//    Albany 3.0:  Copyright 2016 Sandia Corporation               //
+//    This Software is released under the BSD license detailed     //
+//    in the file "license.txt" in the top-level Albany directory  //
+//*****************************************************************//
+
+#ifndef ALBANY_UNIVARIATEDISTRIBUTION_HPP
+#define ALBANY_UNIVARIATEDISTRIBUTION_HPP
+
+#include <boost/math/special_functions/erf.hpp>
+
+namespace Albany
```

Comment:
Since this is not an evaluator (and not in PHAL namespace), maybe the file should be called Albany_UnivariateDistribution.hpp, and perhaps put in `src/utility`, rather than the evaluators folder?